nc-config.cmake.in

```diff
@@ -73,7 +73,7 @@ else
     has_hdf5="yes"
 fi
 
-has_szlib="@USE_SZLIB@"
+has_szlib="@USE_SZIP@"
```

Comment:
This was the *only* instance of `USE_SZLIB` in the entire NetCDF code base, so there was no way that it could be set unless I am overlooking something...  It is also the symbol that is used for the `has_szlib` setting in `libnetcdf.settings`